README.md

```diff
@@ -94,9 +94,15 @@ scrape_configs:
 The ICMP probe requires elevated privileges to function:
 
 * *Windows*: Administrator privileges are required.
-* *Linux*: root user _or_ `CAP_NET_RAW` capability is required.
-  * Can be set by executing `setcap cap_net_raw+ep blackbox_exporter`
-* *BSD / OS X*: root user is required.
+* *Linux*: either the root user, the `CAP_NET_RAW` capability or a user with a
```

Comment:
Let's document these in order of "best practice". So mention the rootless option first, CAP_NET_RAW second, and root user third.

Reply:
Rearranged.